CHANGELOG.md

```diff
@@ -9,7 +9,8 @@
   code blocks will be ignored.
 * Added option `:ignore_code_blocks` to rule MD013. If set to true, hard tabs in
   code blocks will be ignored. The option `:code_blocks` has been marked as
-  deprecated in the documentation.
+  deprecated in the documentation. If `:code_blocks` is set to true in the
```

Comment:
actually, its printed if it's set to `false`. :)

Reply:
You are right. Luckily, we opted to use a more descriptive name for this parameter. :wink: 

I updated the changelog accordingly.